src/plugins/tabs/components/tabs.vue

```diff
@@ -158,6 +171,27 @@ export default {
 
             this.loadedTabs[tab.keyString] = true;
         },
+        getTabStyles(tab) {
+            let styles = {};
+
+            if (!this.isCurrent(tab)) {
+                styles = {
+                    height: this.tabHeight,
+                    width: this.tabWidth
+                };
+            }
+
+            return styles;
+        },
+        getTabWidthAndHeight() {
+            const tabsViewEl = document.getElementsByClassName('c-tabs-view')[0];
```

Comment:
This element already has a ref, let's use it instead of finding the element again on each resize event.